### PR TITLE
Add new selenium test case to test clear output button

### PR DESCRIPTION
**Pull request by ravicodder**
##What is this PR for?

Add a new test case for testing the clear output button.
Clear output button clears the output on the paragraph

##What type of PR is it?

Test

##Is there a relevant Jira issue?

NA

##How should this be tested?

On OSX, you'll need firefox 42.0 installed, then you can run with

PATH=~/Applications/Firefox.app/Contents/MacOS/:$PATH CI="" \
mvn -Dtest=org.apache.zeppelin.ParagraphActionsIT -Denforcer.skip=true \
test -pl zeppelin-server

##Questions:

Does the licenses files need update?NO
Is there breaking changes for older versions?NO
Does this needs documentation?NO

### CODE REVIEW

File: zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java [hunk 1]
```diff
@@ -55,7 +56,38 @@ public void tearDown() {
...
+      return;
+    }
+    try {
+      createNewNote();
 
+      waitForParagraph(1, "READY");
+      WebElement paragraph1Editor = driver.findElement(By.xpath(getParagraphXPath(1) + "//textarea"));
+      paragraph1Editor.sendKeys("println" + Keys.chord(Keys.SHIFT, "9") + "\""
+              + "abcd\")");
+      collector.checkThat("The Output field contains contains",
```
Comment (prabhjyotsingh) [inline zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: contains contains ?

File: zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java [hunk 2]
```diff
@@ -55,7 +56,38 @@ public void tearDown() {
...
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class,'text ng-scope')]")).getText(),
+              CoreMatchers.equalTo(""));
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@ng-click='runParagraph(getEditorValue())']")).click();
+      collector.checkThat("The Output field contains contains ",
```
Comment (prabhjyotsingh) [inline zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: same here "contains contains"

**Comment by prabhjyotsingh** [comment]
Tested, CI green, LGTM.

**Comment by corneadoug** [comment]
CI is now failing on:
```
Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 25.538 sec <<< FAILURE! - in org.apache.zeppelin.integration.ParagraphActionsIT
testClearOutputButton(org.apache.zeppelin.integration.ParagraphActionsIT)  Time elapsed: 5.039 sec  <<< ERROR!
org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[@ng-controller=\"ParagraphCtrl\"][1]//div[contains(@class,'text ng-scope')]"}
```

**Comment by prabhjyotsingh** [comment]
This should catch all Exception not just ElementNotVisibleException.

**Comment by ravicodder** [comment]
@prabhjyotsingh  @corneadoug  
Changes has been done, Please have a look.

**Comment by ravicodder** [comment]
@prabhjyotsingh used handleException, please have a look

**Comment by corneadoug** [comment]
Question, why is that test deleting: `testCreateNewButton`? Is that a result of the rebase?

**Comment by ravicodder** [comment]
@corneadoug Corrected bad rebase.

File: zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java [hunk 3]
```diff
@@ -269,4 +269,38 @@ public void testDisableParagraphRunButton() throws Exception {
...
+    }
+    try {
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+      WebElement paragraph1Editor = driver.findElement(By.xpath(getParagraphXPath(1) + "//textarea"));
+      paragraph1Editor.sendKeys("println" + Keys.chord(Keys.SHIFT, "9") + "\""
+          + "abcd\")");
+      collector.checkThat("Before Run Output field contains ",
+          driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@ng-if,'getResultType()')]")).getText(),
```
Comment (corneadoug) [inline zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: Same reusable xpath advice than in the other PR.
Maybe after those are merged, you could create a PR to apply it in all the tests

Comment (ravicodder) [inline zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: @corneadoug Taken care of it.

**Comment by corneadoug** [comment]
LGTM

**Pull request closed without merging**